tests/IdentifyServiceProviderTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Unicodeveloper\Identify\Test;
+
+use Mockery\MockInterface;
+use PHPUnit_Framework_TestCase;
+use Unicodeveloper\Identify\IdentifyServiceProvider;
+
+class IdentifyServiceProviderTest extends PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var MockInterface
+     */
+    protected $mockApp;
+
+    /**
+     * @var IdentifyServiceProvider
+     */
+    protected $provider;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->mockApp = \Mockery::mock();
```

Comment:
I am still a bit worried here tho, It's a good thing that you are creating an anonymous mock, but I know you should still bind the anonymous mock to a class or dependency. In this file, it's not bound to any, so how does that work? @perfectmak